frontend/__tests__/components/factory/list-page.spec.tsx

```diff
@@ -22,11 +23,11 @@ describe(TextFilter.displayName, () => {
   });
 
   it('renders text input', () => {
-    const input: ShallowWrapper<React.InputHTMLAttributes<any>> = wrapper.find('input');
+    const input: ShallowWrapper<any> = wrapper.find(TextInput);
 
     expect(input.props().type).toEqual('text');
     expect(input.props().placeholder).toEqual(`Filter ${label}...`);
-    expect(input.props().onChange).toEqual(onChange);
+    expect(input.props().onChange()).toEqual(((e) => onChange(e))());
```

Comment:
Is this change still needed? I thought we went back to `onChange={onChange}`.

Reply:
Not anymore.